app/src/main/java/com/firebase/uidemo/auth/AuthUiActivity.java

```diff
@@ -120,6 +120,9 @@
     @BindView(R.id.google_scope_games)
     CheckBox mGoogleScopeGames;
 
+    @BindView(R.id.allow_new_email_accounts)
+    CheckBox mAllowNewEmailAccounts;
```

Comment:
Small nit, this should be under `mEnableSmartLock`. To be honest, all of these should be reorganized in the order they appear in the sample so I guess it doesn't really matter.